src/l1-resources/project/index.ts

```diff
@@ -68,6 +61,11 @@ export interface CfnProjectProps {
    * @schema CfnProjectProps#RegionUsageRestrictions
    */
   readonly regionUsageRestrictions?: string;
+
+  /**
+   * @schema CfnProjectProps#Tags
+   */
+  readonly tags?: any;
```

Comment:
I imagine the any type is simply the best approximation CDK has for when we use untyped object in CFN schema right?